app/routes/pages/components/DisplayVision.js

```diff
@@ -0,0 +1,153 @@
+import React from 'react';
+import styles from './DisplayVision.css';
+import cx from 'classnames';
+import Vision from 'app/components/Vision';
+import DividerWithDots from 'app/components/DividerWithDots';
+
+const DisplayVision = ({ title, vision1, vision2, vision3, vision4 }) => {
+  return (
```

Comment:
When you are returning directly within the brackets, you can rewrite it to ` => (` and avoid the `return (` line :)